### ISSUE TITLE
Regarding Sentiment Analysis

### ISSUE THREAD

**Issue by akhilravoori** [alexanderpei/DeepEarnings#2]
Have you considered using a CNN for sentiment analysis as a binary classifier instead of a multilayer perceptron? And I'm interested in contributing to this project but I don't know how to get in touch with you, could you maybe drop your email id? Thanks

**Comment by alexanderpei** [comment]
Absolutely, that may be a good idea. You can reach me at alexanderpei6@gmail.com
